RealmSwift/Map.swift

```diff
@@ -0,0 +1,332 @@
+//
+//  File.swift
+//  RealmSwift
+//
+//  Created by Pavel Yakimenko on 27/01/2021.
+//  Copyright © 2021 Realm. All rights reserved.
+//
+
+import Foundation
+import Realm
+import Realm.Private
+
+/**
+ * Key-value collection. Where the key is a string and value is one of the available Realm types.
+ * We use Map to don't intefere with the native Swift's Dictionary type.
+ */
+public final class Map<Element: RealmCollectionValue>: RLMDictionaryBase {
```

Comment:
`public subscript(key: Key) -> Value?` is missing. This subscript will need to handle `get` and `set`

Reply:
`public var count: Int { return Int(_rlmCollection.count) }` is missing

Reply:
thanks, fixed